### DISCUSSION TITLE
unable to update my mod to 1.19.3

### DISCUSSION THREAD

**Discussion by mohayyin** [FabricMC/fabric-api#2753] [Mod Dev Support]
hello, there, im new to modding but i have a mod that i started making from 1.19.2 and im struggling to update it to 1.19.3. i did update the gradle.properties file but still doesnt work.
any help?
Yes, @12122234  is my PC's account

**Comment by oliviathevampire** [comment]
Everything related to registries have changed, there is also resource pack changes and lots of other changes in 1.19.3 which you need to port to.

**Reply by 12122234** [reply]
This is my PC's account: I know those changes, but it just won't run 1.19.3, it will run 1.19.2 instead
